server/gitrest/packages/gitrest-base/src/utils/isomorphicgitManager.ts

```diff
@@ -442,7 +448,21 @@ export class IsomorphicGitManagerFactory extends RepositoryManagerFactoryBase<vo
 	 *
 	 * This brings file reads from 1 to 0, and writes from 10 to 3.
 	 */
-	private async slimInit(fs: IFileSystemManager, gitdir: string): Promise<void> {
+	private async slimInit(
+		fs: IFileSystemManager,
+		gitdir: string,
+		fsParams: IFileSystemManagerParams | undefined,
+	): Promise<void> {
+		const simplifiedCustomData = fsParams?.simplifiedCustomData;
+		if (simplifiedCustomData) {
+			// Only pass valid simplifiedCustomData to fs mkdir call during slimInit
```

Comment:
This comment implies simplifiedCustomData should be passed into all of the mkdir calls within `slimInit`, but it's only being passed into the "root". I think this also will add an additional mkdir call, which adds latency (possibly significant depending on storage location). We have tests that prevent adding additional storage calls, so those might fail.

Do we absolutely need to add an extra call here? Or can this data be sent with the other 2 mkdir calls instead?

Reply:
OK great, thanks for confirming!